config/passport/passport.google.config.js

```diff
@@ -0,0 +1,75 @@
+// Package imports
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+const env = require('dotenv').config();
+const to = require('await-to-js').default;
+
+// Constant imports
+const constants = require('../../constants/constants');
+const googleConstants = require('../constants');
+
+// Model imports
+const User = require('../../modules/auth/auth.model');
+
+// Logger import
+const logger = require('../logger/logger');
+
+const passportGoogleConfig = (passport) => {
```

Comment:
Function level comment missing. Please add function level comments to all your functions.

Reply:
Done